rbac/migration_tool/migrate.py

```diff
@@ -89,11 +89,22 @@ def migrate_role(
 
 def migrate_workspace(tenant: Tenant, write_relationships: bool):
     """Migrate a workspace from v1 to v2."""
-    root_workspace = f"root-workspace-{tenant.org_id}"
-    # Org id represents the default workspace for now
+    root_workspace, _ = Workspace.objects.get_or_create(tenant=tenant, type=Workspace.Types.ROOT)
+    default_workspace, _ = Workspace.objects.get_or_create(
+        tenant=tenant, type=Workspace.Types.DEFAULT, parent=root_workspace
+    )
```

Comment:
```suggestion
    root_workspace, _ = Workspace.objects.get_or_create(
        tenant=tenant, type=Workspace.Types.ROOT, name=tenant.tenant_name
    )
    default_workspace, _ = Workspace.objects.get_or_create(
        tenant=tenant, type=Workspace.Types.DEFAULT, parent=root_workspace, name="Default Workspace"
```

Looks like we need to add the names here